experimental/PropertyDDS/packages/property-common/.eslintrc.cjs

```diff
@@ -12,6 +12,7 @@ module.exports = {
 		project: ["./tsconfig.json", "./src/test/tsconfig.json"],
 	},
 	rules: {
+		"@fluid-internal/fluid/no-unchecked-record-access": "warn",
 		"prefer-arrow-callback": "off",
```

Comment:
Re-issue post conflict resolution:
Is this and similar changes needed? @RishhiB, did some needed changes get missed in your update? Or is this PR hitting more than required?

Reply:
In my testing, all of them have been necessary to pass CI. There might have been new code introduced or they somehow got missed. @RishhiB is pulling the config changes out into a separate PR. I'll mark this as a draft in the meantime.